assets/styles/style.css

```diff
@@ -360,3 +360,35 @@ PLUGIN STYLES
   padding-left: 30px;
   background-position: 3px 4px;
 }
+
+.menu-item.members.active > a,
+.dropdown .menu-item.members > a {
+  background-image: url(/wp-content/themes/bfcom/assets/images/members.svg);
+  background-repeat: no-repeat;
+  padding-left: 30px;
+  background-position: 3px 4px;
+}
+
+.menu-item.groups.active > a,
+.dropdown .menu-item.groups > a {
+  background-image: url(/wp-content/themes/bfcom/assets/images/groups.svg);
+  background-repeat: no-repeat;
+  padding-left: 30px;
+  background-position: 3px 4px;
+}
+
+.menu-item.active > a,
```

Comment:
You need to include  `menu-item-167` here; otherwise it's not specific enough and it also styles the home menu:

<img width="530" alt="image" src="https://user-images.githubusercontent.com/338989/45985202-0035ab00-c01a-11e8-85f4-c75dc66da15e.png">

Other than that, this patch is looking good!